scripts/linters/pylint_extensions.py

```diff
@@ -1782,6 +1779,50 @@ def visit_classdef(self, node):
             self.add_message('newline-below-class-docstring', node=node)
 
 
+class SingleLinePragmaChecker(checkers.BaseChecker):
+    """Custom pylint checker which checks if pylint pragma is used to disable
+    a rule for a single line only.
+    """
+
+    __implements__ = interfaces.ITokenChecker
+
+    name = 'single-line-pragma'
+    priority = -1
+    msgs = {
+        'C0027': (
+            'Pylint pragmas should be used to disable a rule '
+            'for a single line only',
+            'single-line-pragma',
+            'Please use pylint pragmas to disable a rule for a single line only'
+        )
+    }
+
+    def process_tokens(self, tokens):
+        """Custom pylint checker which allows paramas to disable a rule for a
+        single line only.
+
+        Args:
+            tokens: Token. Object to access all tokens of a module.
+        """
+        for (token_type, _, (line_num, _), _, line) in tokens:
+            if token_type == tokenize.COMMENT:
+                line = line.lstrip()
+                # Ignore line that is enabling this check.
+                # Example:
+                # # pylint: disable=import-only-modules, single-line-pragma
+                # def func(a, b):
+                # # pylint: enable=import-only-modules, single-line-pragma
+                # Now if do not ignore the line with 'enable' statement
+                # pylint will raise the error of single-line-pragma because
+                # from here on all this lint check is enabled. So we need to
+                # ignore this line.
+                if line.startswith(b'# pylint:'):
```

Comment:
Do we allow `# <something> pylint: disable/enable`? (I see `# isort=skip pylint: disable`, so wanted to check!) If yes then we should not expect the line to start with # pylint? (not sure!)

Reply:
I think how you have it is OK, but I wonder if the check on this line is too narrow. Is it worth catching cases like `#pylint` and `#   pylint` as well?

(If so, please do this in a separate follow-up PR.)

Reply:
We already have plans to check for correct format of comments, i.e. if every comment has a single space after `#`. That will solve this issue automatically.

Reply:
But is this check in place yet? If not, then please still fix this.

Reply:
Ok, I will create a follow-up PR to fix this by today.

Reply:
Created the PR https://github.com/oppia/oppia/pull/10050 to make the check broader and check if there are no or more than one space between `#` and `pylint:`. Example: `#pylint:`, `#          pylint:`